chore(add retrain button and collect data only first time)

mohsinayaz opened the pull request:
Fixes #21  #22 

1:- on thr first time collecting the data we are set a modal-files variable to the local storage and then navigate to the home page. When user close the app and re-open the app it checks if the model-files variable exist in the local Storage it will be redirect to the home page not the collecting page.
2:- add retrain button on the toolbar of header when user click on it will be redirect to the collecting page and remove the local Storage modal-files variable

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have run tests (npm run test & npm run e2e) that prove my fix is effective or that my feature works

AhsanAyaz reviewed (approved):
LGTM 👍🏼 Minor improvements required but we can talk about them later!

mohsinayaz committed:
chore(add retrain button and collect data only first time)

1:- on thr first time collecting the data we are set a modal-files variable to the local storage and then navigate to the home page. When user close the app and re-open the app it checks if the model-files variable exist in the local Storage it will be redirect to the home page not the collecting page.
2:- add retrain button on the toolbar of header when user click on it will be redirect to the collecting page and remove the local Storage modal-files variable

mohsinayaz committed:
fix(header component unit test)
